integrations/GA4/utils.js

```diff
@@ -236,11 +237,8 @@ const getGa4SessionId = (measurementId) => {
     sessionId = Cookie.get(`_ga_${measurementIdArr[1]}`).split(".");
     return sessionId[2];
   }
-  if (
-    window.localStorage &&
-    window.localStorage.getItem(`_ga_${measurementIdArr[1]}`)
-  ) {
-    sessionId = window.localStorage.getItem(`_ga_${measurementIdArr[1]}`);
+  if (Store.get(`_ga_${measurementIdArr[1]}`)) {
+    sessionId = Store.get(`_ga_${measurementIdArr[1]}`);
```

Comment:
Both from Cookie and Local Storage modules read only once!

Reply:
updated